### ISSUE TITLE
Configuration of the default execution strategy

### ISSUE THREAD

**Issue by Grummfy** [celery/celery#8337]
# Checklist

- [x] I have checked the [issues list](https://github.com/celery/celery/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Issue+Type%3A+Feature+Request%22+)
  for similar or identical feature requests.
- [x] I have checked the [pull requests list](https://github.com/celery/celery/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3A%22PR+Type%3A+Feature%22+)
  for existing proposed implementations of this feature.
- [x] I have checked the [commit log](https://github.com/celery/celery/commits/main)
  to find out if the same feature was already implemented in the
  main branch.
- [x] I have included all related issues and possible duplicate issues
  in this issue (If there are none, check this box anyway).

#### Related Issues

- None

#### Possible Duplicates

- None

# Brief Summary
I would like to customize the default task execution strategy globally. I think, there is no way to do it today (or I miss it).

So basically, replace the celery.worker.strategy.default with a custom class

# Design

## Architectural Considerations
None

## Proposed Behavior
We can configure the task execution strategy and it change it

## Diagrams
N/A

## Alternatives
None

**Comment by PlugaruT** [comment]
@Grummfy any luck with this? I am also trying to find ways to overwrite the default strategy.

**Comment by auvipy** [comment]
we will need more input and discussion here to move forward....

**Comment by PlugaruT** [comment]
It's about being able to overwrite the task strategy from here https://github.com/celery/celery/blob/main/celery/app/task.py#L180.
Currently, I was not able to do it as the Worker crashes.
